### PR TITLE
te-13.1: update readme according to the test code

### PR DESCRIPTION
**Pull request by LimeHat**
- README clarifications according to the discussion in #1280 and the code
- getRPC -> Get RPC

---
<sup>
This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia’s intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind.
</sup>

### CODE REVIEW

File: feature/experimental/gribi/ate_tests/route_addition_during_failover_test/README.md [hunk 1]
```diff
@@ -16,22 +16,24 @@ Validate gRIBI route persistence during SSO
...
-*   Validate that the entries are installed as FIB_PROGRAMMED using getRPC.
+*   Validate that the entries are installed as FIB_PROGRAMMED using Get RPC.
 
-*   Send traffic from ATE port-1 to prefixes in IPBlock1 and ensure traffic flows 100% and reaches ATE port-2.
+*   Send traffic from ATE port-1 to prefixes in IPBlock1, ensure traffic flows 100% and reaches ATE port-2, stop the traffic.
 
 *   Start injecting another 1000 IPv4Entries(`IPBlock2: 198.18.100.1/22`) in default VRF with NHGID: #1. 
 
-*   Check for coredumps in the DUT and validate that none are present.
+*   Check for gRIBI coredumps in the DUT and validate that none are present.
```
Comment (xw-g) [inline feature/experimental/gribi/ate_tests/route_addition_during_failover_test/README.md:25]: Can you help add a TODO here about checking not just gRIBI core-dumps? We prefer to check for any coredumps.

File: feature/experimental/gribi/ate_tests/route_addition_during_failover_test/README.md [hunk 2]
```diff
@@ -16,22 +16,24 @@ Validate gRIBI route persistence during SSO
...
 
-*   Concurrently, trigger a supervisor switchover using gNOI `SwitchControlProcessor`  while `IPBlock2` entries are only partially installed.
+*   Concurrently, trigger a supervisor switchover using gNOI `SwitchControlProcessor` while `IPBlock2` entries are only partially installed.
 
-*   Check for coredumps in the DUT and validate that none are present post failover 
+*   Check for gRIBI coredumps in the DUT and validate that none are present post failover
```
Comment (xw-g) [inline feature/experimental/gribi/ate_tests/route_addition_during_failover_test/README.md:29]: Same as above.

**Comment by xw-g** [comment]
Looks good, just some minor comment. Once it's addressed, we can merge. Thanks!!!

**Comment by LimeHat** [comment]
Thanks for the review;
updated, please take a look.

**Review by xw-g — APPROVED** [review]

**Pull request merged**